paper-ripple-behavior.html

```diff
@@ -77,11 +77,13 @@
         if (rippleContainer) {
           Polymer.dom(rippleContainer).appendChild(this._ripple);
         }
-        var domContainer = rippleContainer === this.shadyRoot ? this :
-          rippleContainer;
         if (opt_triggeringEvent) {
```

Comment:
This is kind of orthogonal, but the identifier naming here needs cleanup (`opt_triggeringEvent` -> `optTriggeringEvent`)